backend/libraries/types/src/notifications.rs

```diff
@@ -223,5 +223,5 @@ fn notification_length() {
 
     let bytes = candid::encode_one(notification).unwrap().len();
 
-    assert!(bytes < 630, "{bytes}");
+    assert!(bytes < 850, "{bytes}");
```

Comment:
It's growing...

Reply:
Yeah... not ideal. We'll fix it properly at some point.